static/src/javascripts/projects/common/modules/identity/modules/modal.js

```diff
@@ -4,6 +4,35 @@ import fastdom from 'lib/fastdom-promise';
 const ERR_MODAL_NOT_FOUND = 'Modal not found';
 const ERR_MODAL_MALFORMED = 'Modal is malformed';
 
+const removeNewsletterReminders = (modalEl: HTMLElement) => {
```

Comment:
you can get away with never removing them by cleaning the contents whenever you use `insertNewsletterReminders` instead of being checking for closes